youtube_dl/extractor/yahoo.py

```diff
@@ -151,6 +163,10 @@ def _real_extract(self, url):
                 items = json.loads(items_json)
                 video_id = items[0]['id']
                 return self._get_info(video_id, display_id, webpage)
+        # Look for NBCSports iframes
+        nbc_sports_url = NBCSportsVPlayerIE._extract_url(webpage)
+        if nbc_sports_url:
+            return self.url_result(nbc_sports_url, 'NBCSportsVPlayer')
```

Comment:
This should be placed in generic extractor as well. vplayer is a generic embed (e.g. http://bbs.clutchfans.net/showthread.php?t=244180)